test/await-use.test.js

```diff
@@ -270,3 +270,25 @@ test('await use - mix of same and future tick callbacks', async (t) => {
   record += 'ready'
   t.equal(record, 'plugin0|plugin1|plugin2|ready')
 })
+
+test('await use - fork the promise chain', { only: true }, (t) => {
```

Comment:
What does only:true mean?

Reply:
It should be removed, it tells to tap to **only** run this test.

Reply:
Nothing. If it's run with TAP_ONLY=true env variable, it would run only this test. I'll remove, good spot!